packages/nimble-components/src/wafer-map/modules/event-coordinator.ts

```diff
@@ -11,13 +11,15 @@ export interface EventCoordinatorCallbacks {
  * EventCoordinator deals with user interactions and events
  */
 export class EventCoordinator {
-    private readonly zoomHandler: ZoomHandler;
-    private readonly hoverHandler: HoverHandler;
-
+    private readonly zoomHandler;
+    private readonly hoverHandler;
     public constructor(private readonly wafermap: WaferMap) {
         this.zoomHandler = new ZoomHandler(wafermap);
         this.hoverHandler = new HoverHandler(wafermap);
+    }
 
+    public updateEvents(): void {
```

Comment:
Can you instead merge attachEvents and updateEvents into a single public function called `attachEvents`?

```ts
    public attachEvents(): void {
        this.zoomHandler.updateZoomBehavior();
        this.wafermap.addEventListener('mousemove', this.onMouseMove);
        this.wafermap.addEventListener('mouseout', this.onMouseOut);
        this.wafermap.canvas.addEventListener('wheel', this.onWheelMove, {
            passive: false
        });
    }
```

That way the public api is `attachEvents` and `detachEvents`.

I recognize other changes are more of an update pattern but this case is more of a attach / create